src/consts/gasTankFeeTokens.ts

```diff
@@ -832,6 +839,13 @@ export default [
     decimals: 18,
     icon: 'https://assets.coingecko.com/coins/images/16547/standard/arb.jpg?1721358242'
   },
+  {
+    chainId: 42161n,
+    address: '0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9',
+    symbol: 'USD₮0',
```

Comment:
All others have their symbols in lowercase. I'm not sure if we use symbol anywhere, but for consistency, we may want to normalize the symbol casing of all of them, not only for USD₮0?